src/validations/addNewUserSchema.ts

```diff
@@ -0,0 +1,34 @@
+import parsePhoneNumber from 'libphonenumber-js';
+import { z } from 'zod';
+
+export const addNewUserSchema = z.object({
+  organization_id: z.string({
+    invalid_type_error: 'Field is required.',
+  }),
+  email: z.string().email('Must be correct email format'),
+  first_name: z.string().min(1, 'Field is required.'),
+  last_name: z.string().min(1, 'Field is required.'),
+  role: z.string({
```

Comment:
could we make `z.enum()` here to avoid assertion type `TypeUserIn` in `postUsers` action ?